src/iterator.ts

```diff
@@ -39,7 +39,7 @@ export enum SchemaTypesToIterate {
 
 export type TraverseOptions = {
   callback: TraverseCallback
-  schemaTypesToIterate: SchemaTypesToIterate[]
+  schemaTypesToIterate: Array<`${SchemaTypesToIterate}`>;
```

Comment:
I'm lost on this 😆 Is this the preferable way? Or it was a bug?

Reply:
It's an interpolation of enum values as union. Feature in the TS 4.1, so you can put `schemaTypesToIterate` as array of normal string (but with values from enum) or `SchemaTypesToIterate` signature for given enum value.